Sources/ArcGISToolkit/Components/Form/FormInputs/FormInputModel.swift

```diff
@@ -18,15 +18,18 @@ import SwiftUI
 /// A model for an input in a form.
 ///
 /// - Since: 200.3
-public class FormInputModel: ObservableObject {
+class FormInputModel: ObservableObject {
     /// A Boolean value indicating whether a value in the input is required.
     @Published var isRequired: Bool
     
     /// A Boolean value indicating whether a value in the input is editable.
     @Published var isEditable: Bool
     
     /// The value of the input.
-    @Published var value: String
+    @Published var value: Any?
+    
+    /// The formatted value of the input.
+    @Published var formattedValue: String
```

Comment:
`value` is now Any?. Because `Any` is not comparable, I've added a published `formattedValue: String` property. Inputs can instead now monitor this for changes.

Reply:
Are `value` and `formattedValue` guaranteed to be in sync with each other? I would think so, so this should be fine.

Reply:
So because `Any?` is not `Comparable`, you can't do `.onChange(of: inputModel.value)`?

Reply:
That's correct.